src/pages/OnboardEngagement/PurposeForUsingExpensifyPage.tsx

```diff
@@ -76,7 +76,7 @@ function PurposeForUsingExpensifyModal() {
     const styles = useThemeStyles();
     const {windowHeight} = useWindowDimensions();
     const theme = useTheme();
-    const backgroundColorStyle = StyleUtils.getBackgroundColorStyle(theme.PAGE_THEMES[SCREENS.SETTINGS.WORKSPACES].backgroundColor);
+    const backgroundColorStyle = StyleUtils.getBackgroundColorStyle(theme.PAGE_THEMES[SCREENS.ONBOARD_ENGAGEMENT.ROOT].backgroundColor);
     const appBGColor = StyleUtils.getBackgroundColorStyle(theme.appBG);
 
     const navigateBack = useCallback(() => {
```

Comment:
Please do this instead 
```ts
const navigateBack = useCallback(() => {
        Report.dismissEngagementModal();
        Navigation.goBack();
}, []);
```

Reply:
@MonilBhavsar This will fix https://github.com/Expensify/App/pull/36144#issuecomment-1934190940

Reply:
Applying this. This worked, thanks